mindjunkies/dashboard/tests/test_views.py

```
@@ -158,3 +302,54 @@ def test_verification_wait_view(client, user):
+
+
+@pytest.mark.django_db
+def test_archive_view(client, teacher_user, archived_course):
+    client.force_login(teacher_user)
+
+    url = reverse("archive_content")
+    response = client.get(url)
+
+    assert response.status_code == 200
+    assert "courses" in response.context
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)